shark/shark-cli/build.gradle

```diff
@@ -15,6 +15,28 @@ tasks.withType(KotlinCompile).configureEach {
   kotlinOptions.jvmTarget = JavaVersion.VERSION_1_8.toString()
 }
 
+jar {
```

Comment:
Can you rename this `uberJar` or `superJar` or `jarWithTransitiveDependencies` ? Something that implies it's not your standard jar but it's a jar with everything (let's not call it "fat jar", that wouldn't feel right)